src/libs/UserService/index.ts

```diff
@@ -1,3 +1,17 @@
 import { Users } from '../../database/models';
 
-export const getUser = async (id) => await Users.findByPk(id).then(result => result.get({plain: true}))
\ No newline at end of file
+export const getUser = async (id) => await Users.findByPk(id).then(result => result.get({plain: true}))
+
+export const getUserFirstAndLastName = async (id) => {
+    const first_name = await Users.findAll({
+        where: { id },
+    }).then(result => result[0].first_name)
+
+        const last_name = Users.findAll({
+        where: { id },
+    }).then(result => result[0].lastName);
```

Comment:
```suggestion
        const last_name = Users. findByPk(id).then(result => result.last_name);
```
Similar suggestion to above. I do notice the lastName property being returned as camelCase instead of snake case. Please check the model to make sure that the type is being returned this way